ui/src/app/components/object-detail/object-detail.component.ts

```diff
@@ -314,6 +314,7 @@ export class ObjectDetailComponent implements OnInit {
             data: { message: res, type: 'error' },
             maxWidth: '500px',
           })
+          this.isEditMode = true
```

Comment:
Interleaving suggestion doesn't go away if you delete pk column and then fk : https://drive.google.com/file/d/1KZP9ow_RFnGlTl8K2Mq7EK9_RpzJzRqw/view?usp=share_link&resourcekey=0-I4iwAqWiZynbMp3FMOhLsA

Reply:
Updated